docs/proposals/performance.md

```diff
@@ -0,0 +1,21 @@
+# Post-1.0 performance/scalability improvement ideas
+
```

Comment:
No. People should feel free to add ideas that relate to any part of the system. I think we should keep it in a single list unless it gets so long that dividing it becomes necessary.